scripts/run-tests.sh

```diff
@@ -81,6 +81,12 @@ retval=$(($retval+$?))
     --gtest_output=xml:softwarecontainerlib_unittest_result.xml
 retval=$(($retval+$?))
 
+if [ $retval -ne 0 ]; then
+	echo "********************************"
+	echo "*** ONE OR MORE TESTS FAILED ***"
+	echo "********************************"
```

Comment:
The point of having this is that since we have many tests in different test suites, there is no way to get output about that some early test failed unless you check the exit status of the test runner. This is pure developer convenience imo.